src/app/admin/admin-reports/filtered-items/filtered-items.component.scss

```diff
@@ -0,0 +1,19 @@
+.predicate {
+    height: 26px;
+}
+
+.num {
+    text-align: center;
+}
+
+.stats tbody tr:nth-child(even) {
+    background: #CCC
+}
+.stats tbody tr:nth-child(odd) {
+    background: #FFF
+}
+
+.stats thead tr {
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+}
```

Comment:
Same comments about bootstrap tables as above

Reply:
Should be OK.

Reply:
Done.